# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/compiler/translator/tree_ops/EmulateGLDrawID.h
# Commit: 13282d7a47a5
# Full Hash: 13282d7a47a57a73be8d060efd9645b4d642f03e
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-09-08 09:46:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 3064469c073d -r 13282d7a47a5 gfx/angle/checkout/src/compiler/translator/tree_ops/EmulateGLDrawID.h
--- a/gfx/angle/checkout/src/compiler/translator/tree_ops/EmulateGLDrawID.h	Sat Sep 07 13:36:46 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,31 +0,0 @@
-//
-// Copyright 2018 The ANGLE Project Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-//
-// EmulateGLDrawID is an AST traverser to convert the gl_DrawID builtin
-// to a uniform int
-//
-
-#ifndef COMPILER_TRANSLATOR_TREEOPS_EMULATEGLDRAWID_H_
-#define COMPILER_TRANSLATOR_TREEOPS_EMULATEGLDRAWID_H_
-
-#include <GLSLANG/ShaderLang.h>
-#include <vector>
-
-#include "compiler/translator/HashNames.h"
-
-namespace sh
-{
-struct Uniform;
-class TIntermBlock;
-class TSymbolTable;
-
-void EmulateGLDrawID(TIntermBlock *root,
-                     TSymbolTable *symbolTable,
-                     std::vector<sh::Uniform> *uniforms,
-                     bool shouldCollect);
-
-}  // namespace sh
-
-#endif  // COMPILER_TRANSLATOR_TREEOPS_EMULATEGLDRAWID_H_